src/scores/stats/novel.py

```diff
@@ -0,0 +1,337 @@
+"""
+This module contains functions for calculating non-standard metrics
```

Comment:
Please move this into `src/scores/continuous` rather than in stats and don't call it "novel", just call it "flipflop".